### PR TITLE
Spring security 5.6.1 upgrade

### PR DESCRIPTION
**Pull request by duranmf**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-2693

### Change description ###

This pr is regarding spring security upgrade to 5.6.1.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by MSancaktutar — APPROVED** [review]
LGTM, but a second review would also be useful.

File: build.gradle [hunk 1]
```diff
@@ -178,16 +178,18 @@ dependencies {
     compile group: 'com.github.hmcts', name: 'idam-java-client', version: '2.0.1'
     compile "org.bouncycastle:bcprov-jdk15on:1.61", "org.bouncycastle:bcpkix-jdk15on:1.61"
 
-    implementation group: 'org.springframework.security', name: 'spring-security-oauth2-client'
+    implementation group: 'org.springframework.security', name: 'spring-security-oauth2-client', version: '5.6.1'
```
Comment (rajaram-nagarajan) [inline build.gradle:181]: As the version is getting repeated in many places, can you move it to ext above and refer the variables here?

Comment (MSancaktutar) [reply build.gradle:181]: Agree.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM with a minor comment

**Pull request merged**